3rdParty/vcpkg_ports/configs/manager/vcpkg.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "boinc-msbuild-x64",
```

Comment:
Is this is for linux, then please change it to `boinc-manager` and put into 'linux' subdirectory

Reply:
done.